src/containers/AdminDashboard/index.js

```diff
@@ -177,9 +195,23 @@ export class AdminDashboard extends Component {
     return true;
   }
 
-  validateDelete() {
+  validateDeleteForum() {
     const { forumId } = this.state;
-    if (forumId !== '') return false;
+    let notEmpty = false;
+    if (forumId !== '') notEmpty = true;
+    if (notEmpty) {
+      return false;
+    }
+    return true;
+  }
+
+  validateDeleteYouth() {
+    const { youthPicsId } = this.state;
+    let notEmpty = false;
+    if (youthPicsId !== '') notEmpty = true;
```

Comment:
@Unesdala you just need to return false here, and get rid of the extra variables please